src/dupradar/dupradar.r

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env Rscript
+
+# Command line argument processing
+args = commandArgs(trailingOnly=TRUE)
+if (length(args) < 5) {
+    stop("Usage: dupRadar.r <input.bam> <sample_id> <annotation.gtf> <strandDirection:0=unstranded/1=forward/2=reverse> <paired/single> <nbThreads> <R-package-location (optional)>", call.=FALSE)
+}
+
+message("paired_end is", args[5])
+message("the type is is", class(args[5]))
+
+input_bam <- args[1]
+output_prefix <- args[2]
+annotation_gtf <- args[3]
+stranded <- as.numeric(args[4])
+paired_end <- ifelse(args[5] == "true", TRUE, FALSE)
+threads <- as.numeric(args[6])
```

Comment:
I wonder if it makes more sense to combine `script.sh` and `dupradar.r` and directly call the R script instead of using the wrapper. But that would force us to _change_ the `dupradar.r` code itself. What do you think?

Reply:
That could be good, it's pretty straightforward and I think it would simplify the component, I'll get on that.